docs/source/install.rst

```diff
@@ -1,20 +1,15 @@
 Installation
 ============
 
-To use Dask-CUDA on your system, you will need the following:
-
-- At least one CUDA-compatible GPU
-- NVIDIA CUDA Toolkit
-- *Anything else?*
-
-Once these requirements are satisfied, Dask-CUDA can be install using ``conda``, ``pip``, or from source.
+To use Dask-CUDA on your system, you will need the `NVIDIA CUDA Toolkit <https://developer.nvidia.com/cuda-toolkit>`_.
```

Comment:
This is not exactly correct either. If you have a driver version compatible with that of `cudatoolkit` package installed via conda, you don't need to install the toolkit from the NVIDIA website. IOW, this should only be needed for pip.

Reply:
Could we address this as well? I think after that we're good to merge.

Reply:
Sure! Sorry, missed this one.

Reply:
Actually, for clarity, is the driver version in question that of CUDA?

Reply:
Driver and CUDA toolkit versions are two different things, the CUDA toolkit version must always be compatible with that of the driver version. Take a look at [Table 1](https://docs.nvidia.com/deploy/cuda-compatibility/index.html#binary-compatibility), it lists what's the minimum driver version you need for a specific CUDA toolkit's version.

Reply:
What I'm trying to understand is how the driver is installed (it seems like it's included with a website download of CUDA toolkit, but I'm not finding much info on downloading the driver itself), and then maybe how to list the currently installed version of the driver. So far, I'm imagining the install section should look something like:

**Conda**
- Install NVIDIA driver or identify the currently installed version
- Conda install Dask-CUDA and a compatible version of CUDA toolkit (maybe linking the chart)

**Pip**
- Install CUDA toolkit from the website (this should come with the NVIDIA driver)
- Pip install Dask-CUDA

Reply:
The driver has to be installed by a user with admin privileges (i.e., root), therefore drivers can't be distributed via conda, so that means the driver is a system requirement. The driver can be obtained from a package of the user's Linux distro, or from NVIDIA's website, and in the latter case it's distributed both as part of the [CUDA toolkit](https://developer.nvidia.com/cuda-toolkit) as a [standalone package](https://www.nvidia.com/Download/index.aspx).

I think the idea above for Conda and Pip looks fine to me.

Reply:
Thanks! I actually found [this installation guide](https://docs.nvidia.com/datacenter/tesla/tesla-installation-notes/index.html), which reiterates what you said - I think I'll include as the primary resource for handling driver support (in the Conda case).